llvm/lib/Target/AMDGPU/AMDGPULegalizerInfo.cpp

```diff
@@ -2764,6 +2768,45 @@ bool AMDGPULegalizerInfo::buildPCRelGlobalAddress(Register DstReg, LLT PtrTy,
   return true;
  }
 
+ // Emit a ABS32_LO / ABS32_HI relocation stub.
+ void AMDGPULegalizerInfo::buildAbsGlobalAddress(
```

Comment:
This function header is indented with a single space. That probably confuses your clang-format.

Reply:
Thanks for the hint. That formatting was introduced by clang-format. The method above had its closing bracket intended with a space which seems to have caused issues for my code.